src/main/groovy/org/labkey/gradle/util/BuildUtils.groovy

```diff
@@ -882,7 +882,7 @@ class BuildUtils
 
     static boolean useEmbeddedTomcat(Project project)
     {
-        project.hasProperty(USE_EMBEDDED_TOMCAT)
+        project.hasProperty(USE_EMBEDDED_TOMCAT) && "false" != project.property(USE_EMBEDDED_TOMCAT)
```

Comment:
You'll want to update the raw usages of `project.hasProperty("useEmbeddedTomcat")` in the root `settings.gradle` file and the bootstrap `build.gradle` file  to use this method as well.

Reply:
`bootstrap/build.gradle` already uses this and `settings.gradle` doesn't have a `Project` to pass to this. I've created an overload that takes a `Settings` and performs a similar calculation if you'd like to re-review.